manifests/controller-manager/cloud-config

```diff
@@ -1,10 +1,8 @@
 [Global]
-region=RegionOne
-username=admin
-password=passw0rd
 auth-url=http://192.128.1.15:5000/v3
```

Comment:
why removed this? authentication by application credentials not used by everyone

Reply:
This is more of a best-practice scenario that goes along with the inclusion of application credentials with PR https://github.com/kubernetes/cloud-provider-openstack/pull/735 that deduplicated auth in the client as well as amended the feature to the provider documentation.

Reply:
I suggest we keep this for historical reasons but suggest users use application credential if possible in the doc, because using application credential would add more configuration steps than before for existing CPO users.

Reply:
agreed